doc/source/how-to/setting-up.rst

```diff
@@ -0,0 +1,528 @@
+
+Verification
+~~~~~~~~~~~~
+Once your installation process is complete, verify your Git installation by
+running:
+
+.. tabs:: 
+
+    .. group-tab:: Windows CMD
+
+        .. code-block:: text
+
+            git --version
+
+    .. group-tab:: Windows PowerShell
+
+        .. code-block:: text
+
+            git --version
+
+    .. group-tab:: macOS
+
+        .. code-block:: text
+
+            git --version
+
+    .. group-tab:: Linux/UNIX
+
+        .. code-block:: text
+
+            git --version
+
+Usage
+~~~~~
+If you are not familiar with Git, see the `Git Reference Manual <https://git-scm.com/doc>`_.
+for comprehensive information on how to use it.
+
+Configuration
+~~~~~~~~~~~~~
+It is very important to properly configure Git so that every modification that you make
+to the code points to you. There are two types of configuration:
+:ref:`Global` and :ref:`Local`. It is also possible to combine both to have
+a :ref:`Dynamic` configuration. 
+
+Global
+++++++
+Global configuration are automatically included in every Git repository on
+your machine unless overridden by a :ref:`Local` configuration, which
+is located in ``C:\Users\<username>\.gitconfig`` for Windows users or in
+``/home/<username>/.gitconfig`` for macOS/Linux/UNIX users.
+
+You can set the value for any variable in a field by running:
+
+.. code-block:: bash
+
+   git config --global <field>.<varname> <value>
+
+Some examples follow.
+
+**Set up your name** 
+
+.. code-block:: bash
+
+    git config --global user.name <Your Name>
+
+**Set up your email** 
+
+.. code-block:: bash
+
+    git config --global user.email <Ansys Email>
+
+**Set up the default branch name** 
+
+.. code-block:: bash
+
+    git config --global init.defaultBranch main
+
+Local
++++++
+Sometimes, you may want to declare a specific configuration to be used only in a
+project of your interest. To override the :ref:`Global` configuration, it is
+possible to declare a local one.
+
+Commands work the same as :ref:`Global` ones except that instead of the
+``--global`` flag you need to use the ``--local`` flag. Make sure you run this
+commands in the root directory of your project and that a ``.git/`` folder
+exists.
+
+If you would like to manually modify your local configuration, this is saved in
+the ``.git/config`` file.
+
+Dynamic
++++++++
+It is possible to configure :ref:`Git` such that it selects between multiple
+configuration profiles according to whether your project is located in your system.
+This allows you to define common configurations for working under ``PyAnsys``,
+``Ansys`` or open source projects from which the company benefits.
+
+As an example, consider the following scenario for setting up two git
+configuration profiles for working with ``Ansys`` and personal projects.
+
+Create two files and name those such that you recognize which :ref:`Git`
+configuration represent. For example, ``.gitconfig-ansys`` and
+``.gitconfig-personal``.
+
+Each one of these files may look like this:
+
```

Comment:
I think the upcoming tabs should go right here, instead of after the next paragraph...

Reply:
You are right, thanks for pointing this out.